tests/core/data_model/test_dataobject.py

```diff
@@ -454,47 +365,68 @@ class Bar(DataObjectBase):
             Annotated[Bar, FieldNumber(2), OneofField("bar")],
         ]
 
-        def __getattr__(self, name):
```

Comment:
All of this custom stuff that needed to be added to support `oneofs` is now implicit